HLTrigger/HLTcore/src/HLTConfigData.cc

```diff
@@ -117,9 +118,17 @@ void HLTConfigData::extract() {
   // Obtain module labels of all modules on all trigger paths
   const unsigned int n(size());
   moduleLabels_.reserve(n);
+  std::unordered_set<std::string> processDirectives = {{"@"}, {"#"}};
```

Comment:
How about asking these from `edm::service::TriggerNamesService::schedulingConstructLabels()`? `HLTrigger/HLTcore` seems to already depend on `FWCore/Framework`.

Reply:
I created a standalone function that all changed classes now use.